projects/collector-v1.md

```diff
@@ -0,0 +1,42 @@
+### Description
+
+Quite a few of our users are happy using a non-v1 Collector, as evidenced by talks delivered by eBay or VTEX. On the other hand, other companies have internal policies forbidding the usage of software that hasn’t reached v1.
+
+Other companies, hearing the success stories of the “early adopters”, tried to use the Collector, but due to a lack of guidance, ended up using components that weren’t ready for prime time and were frustrated.
+
+Our main goal with this project is to define what's the scope for Collector v1, defining which use-cases we want to satisfy first, followed by stabilizing the components that are required for that. Once we are there, we'll focus on documentation for those components and use cases.
+
+We believe we are close to a v1 of the Collector, given that users are using it in production already. Therefore, we are confident we can deliver a v1 this year.
+
+Teams involved in this:
+* @open-telemetry/collector-approvers 
+* @open-telemetry/collector-contrib-approvers 
+* @open-telemetry/collector-maintainers 
+* @open-telemetry/collector-contrib-maintainer 
```

Comment:
Answer can be "no", but to verify: are there an documentation-related goals for v1, so should @open-telemetry/docs-maintainers & @open-telemetry/docs-approvers be involved as well?

Reply:
Docs are definitely part of this and explicitly mentioned in the design doc we have. I'll add those two teams, although I expect the collector teams to do most of the work.